app/api/v2/base.rb

```diff
@@ -32,7 +32,8 @@ class Base < Grape::API
 
     include ExceptionHandlers
 
-    mount Identity::Base   => '/identity'
+    mount Identity::Base => '/identity'
+    mount Auth0::Base => '/auth0'
```

Comment:
I think you dont need to create new module
For identity we have https://github.com/openware/barong/blob/master/app/api/v2/identity/sessions.rb this one
just create a namespace here like auth0


Also do not forget after adding new public endpoint you should add it here https://github.com/openware/barong/blob/master/config/authz_rules.yml